design-documents/graph-ql/coverage/customer/customer-reorder.graphqls

```diff
@@ -1,5 +1,5 @@
 type Mutation {
-    reorderItems(orderId: ID!): ReorderItemsOutput! @doc(description: "Add all items from the registered customer's order to his shopping cart.")
```

Comment:
reorderItems(orderId: ID, orderId: ID) - we will accept both and enforce this through code not schema

Reply:
hi @cpartica, please confirm that this will be implemented and please close this ticket